src/components/navbar/NavLinksPerRole.tsx

```diff
@@ -112,6 +116,10 @@ const navLinksPerRole: NavLinkRoles = {
           },
         ],
       },
+      {
+        title: "",
+        links: [{ title: "", to: "" }],
+      },
```

Comment:
Why add empty objects to the navbar list?

Reply:
fixed